### DISCUSSION TITLE
Request a GHIDRA binary package on the Windows 10 platform

### DISCUSSION THREAD

**Discussion by txgxbj** [NationalSecurityAgency/ghidra#7073] [General]
hello everyone:
I have downloaded the Ghidra binary package multiple times, but it has not been successful. So seek help. Whoever has this document, please send me a copy. Thank you very much. The specific content is as follows:
Ghidra version: 11.12
Platform used: Windows 10
My email: txgxcd@163.com
Rose with people, hands have lingering fragrance, thank you again

**Comment by astrelsky** [comment]
https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_11.2_build/ghidra_11.2_PUBLIC_20240926.zip

**Comment by txgxbj** [comment]
@astrelsky  thank you
